crates/proto/src/version/v662/enums/position_tracking_db_server_broadcast_packet.rs

```diff
@@ -0,0 +1,10 @@
+pub mod PositionTracingDBServerBroadcastPacket {
```

Comment:
It may make sense to also merge this with `position_tracking_db_client_request_packet.rs` into one file called `position_tracking_db.rs`